packages/terra-list/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 4.64.0 - (October 3, 2023)
+
 * Added
   * Added arrow key navigation support for list items.
```

Comment:
I know this was from an older PR but can we fix this and add a newline here?

Reply:
Sure [be9ed02](https://github.com/cerner/terra-core/pull/3928/commits/be9ed0279e4cb4aa3a509a60c0a4930330cc0808)